### PR TITLE
Ond231 500 set language and task in ondewo s2t api

### PR DESCRIPTION
**Pull request by Hannan-Komari**

### CODE REVIEW

File: ondewo/s2t/speech-to-text.proto [hunk 1]
```diff
@@ -104,6 +104,72 @@ enum InferenceBackend {
     INFERENCE_BACKEND_FLAX = 2;
 }
 
+enum Language {
+
+    // Not set
+    UNKNOWN = 0;
+
+    // Multiple language
+    ALL = 1;
```
Comment (teddius) [inline ondewo/s2t/speech-to-text.proto:113]: MULTI instead of ALL

File: ondewo/s2t/speech-to-text.proto [hunk 2]
```diff
@@ -613,25 +699,25 @@ message Wav2VecTriton {
 }
 
 
-// Quartznet contains information about the Quartznet model.
-message Quartznet {
+// // Quartznet contains information about the Quartznet model.
```
Comment (teddius) [inline ondewo/s2t/speech-to-text.proto:702]: Remove code that is not needed

File: ondewo/s2t/speech-to-text.proto [hunk 3]
```diff
@@ -752,24 +838,24 @@ message VoiceActivityDetection {
...
-    string model_path = 1;
+    // Full name of the Pyannote model.
+    string model_name = 1;
 
     // Minimum audio size for processing.
     int64 min_audio_size = 2;
 
-    // Offset for voice activity detection.
-    float offset = 3;
+    // // Offset for voice activity detection.
```
Comment (teddius) [inline ondewo/s2t/speech-to-text.proto:847]: Why can we not set onset and offset anymore?

File: ondewo/s2t/speech-to-text.proto [hunk 4]
```diff
@@ -779,19 +865,19 @@ message Pyannote {
 
 }
 
-// Matchbox contains configuration for the Matchbox voice activity detection model.
-message Matchbox {
+// // Matchbox contains configuration for the Matchbox voice activity detection model.
```
Comment (teddius) [inline ondewo/s2t/speech-to-text.proto:868]: Remove code

**Review by teddius — CHANGES_REQUESTED** [review]

File: ondewo/s2t/speech-to-text.proto (hunk 3 above)
Comment (Hannan-Komari) [reply ondewo/s2t/speech-to-text.proto:847]: Pyannote does not get onset and offset as an input parameter from v3.1.0. (refer to: https://github.com/pyannote/pyannote-audio/releases/tag/3.1.0)

**Review by teddius — APPROVED** [review]

**Pull request merged**